src/pages/content.tsx

```diff
@@ -328,6 +328,7 @@ const Content = () => {
                                 );
                                 setSelectedFileNode(undefined);
                             }}
+                            fileId={selectedFileNode.id}
```

Comment:
I think there might be a merge/version problem here, because I already added a `fileNodeId` property here in a previous PR. Not sure why its not here, might need to integrate with this branch